packages/features/flags/features.repository.ts

```diff
@@ -65,4 +65,15 @@ export class FeaturesRepository implements IFeaturesRepository {
       throw err;
     }
   }
+  async checkIfTeamHasFeature(teamId: number, featureId: keyof AppFlags) {
+    try {
+      const teamFeature = await db.teamFeatures.findUnique({
```

Comment:
Did you mean?

```suggestion
      const teamFeature = await db.teamFeatures.findUniqueOrThrow({
```

Reply:
Not throwing since we're expecting a boolean return value